### PR TITLE
Mero 4 node conf with 4 clients

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Mero 4 node conf with 4 clients

Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-71JT88/
Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by pdhiran — COMMENTED** [review]
Looks good.

File: conf/reef/baremetal/mero_4_node_4_client_conf [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            - /dev/sdi
+            - /dev/sdj
+            - /dev/sdk
+            - /dev/nvme0n1
+            - /dev/sdl
+        - hostname: mero008
+          id: node2
+          ip: 10.8.129.228
+          role:
+            - mds
```
Comment (pdhiran) [inline conf/reef/baremetal/mero_4_node_4_client_conf:38]: Let's remove mds from here.

File: conf/reef/baremetal/mero_4_node_4_client_conf [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            - /dev/sdk
+            - /dev/nvme0n1
+            - /dev/sdl
+        - hostname: mero010
+          id: node3
+          ip: 10.8.129.230
+          role:
+            - mds
+            - mon
+            - rgw
```
Comment (pdhiran) [inline conf/reef/baremetal/mero_4_node_4_client_conf:65]: Let's remove rgw from node3 and node4.

This change is to make sure that we have daemon colocation recommendations met.
https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/6/html/installation_guide/red-hat-ceph-storage-considerations-and-recommendations

**Comment by neha-gangadhar** [comment]
LGTM

**Review by pdhiran — APPROVED** [review]
Looks good

**Pull request merged**